app/src/main/java/com/example/grosharies/ui/theme/Shape.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.grosharies.ui.theme
+
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Shapes
+import androidx.compose.ui.unit.dp
+
+val Shapes = Shapes(
```

Comment:
you only use roundedShapes, but once need for other Shapes comes.. I'd expect Shaped to probably be renamed to roundShapes, xShapes etc.